systers_portal/community/models.py

```diff
@@ -24,13 +24,17 @@ class Community(models.Model):
     parent_community = models.ForeignKey('self', blank=True, null=True,
                                          verbose_name="Parent community")
     website = models.URLField(max_length=255, blank=True,
-                              verbose_name="Website")
+                              verbose_name="Website",
+                              help_text="Enter URL starting with http(s)://")
```

Comment:
Placeholder would be better. You can always raise an issue first, we can discuss there and then you can start with your changes.